sql/core/src/main/scala/org/apache/spark/sql/Dataset.scala

```diff
@@ -597,7 +597,8 @@ class Dataset[T] private[sql](
    * @group basic
    * @since 1.6.0
    */
-  def isLocal: Boolean = logicalPlan.isInstanceOf[LocalRelation]
+  def isLocal: Boolean = logicalPlan.isInstanceOf[LocalRelation] ||
+    logicalPlan.isInstanceOf[CommandResult]
```

Comment:
Could you add a test coverage for this, @beliefer  ?

Reply:
+1, this API has no test as all, we can add one in `DataFrameSuite`

Reply:
Thank you for reminder.